### DISCUSSION TITLE
Tone down Font colour list view files folders

### DISCUSSION THREAD

**Discussion by ketonik** [spacedriveapp/spacedrive#2400] [Feature Request]
So using the dark theme, naturally, the font colour appears to be RGB 255,255,255, I rekon tone it down a bit cos it's very bright and almost creates a halo effect around it, like a bright LED, maybe back it off a touch, perhaps RGB 230.230,230?

I think Apple's Finder is RGB 222,222,222

**Comment by utkubakir** [comment]
Fair, created an issue for this
